content/control-api/curl-examples.textile

```diff
@@ -0,0 +1,205 @@
+---
+title: Examples
+meta_description: "The Control API is a REST API that enables you to manage your Ably account programmatically."
+meta_keywords: "Ably, Control API, account management, control, REST API, OpenAPI Specification, OAS3."
+section: control-api
+index: 20
+jump_to:
+  Help with:
+    - Examples - Apps#examples-apps
+    - Examples - Queues#examples-queues
+    - See also#see-also
+---
+
+This topic includes some examples of using the Control API with "Curl":https://curl.se/.
+
+In the code examples, you will need to set the following variables by any convenient method (such as setting the variables in a script, or copying and pasting suitable values directly into the code):
+
+|_. Variable |_. Description |
+| ACCOUNT_ID | Your Ably Account ID (see "here":/control-api#account-id) |
+| ACCESS_TOKEN | Your Ably access token for the Control API (see "here":/control-api#authentication) |
+| APP_ID | The application ID for the application you want to modify or retrieve information for (see "here":/control-api#app-id) |
+
+h2(#examples-apps). Applications
+
+You can use the Control API to carry out tasks such as listing and creating applications. Operations available include:
```

Comment:
It's slightly jarring to see them referred to as "Applications" here and "apps" elsewhere. Can we link to the glossary definition of an app so that developers know that we're talking about "Ably apps" and not the apps they're trying to build?

Reply:
Yes, it should be "applications". I've changed this throughout. Slightly confusing as the endpoints use 'apps'. But I think "applications" is correct. I've searched for apps and changed to applications, unless it's an endpoint of course.